src/Database/DBA.php

```diff
@@ -1507,6 +1507,15 @@ public static function buildCondition(array &$condition = [])
 	 */
 	public static function buildParameter(array $params = [])
 	{
+		$groupby_string = '';
```

Comment:
@annando just to be sure: Is this grouping OK for you? I checked the output and it seems like it does it right..

Reply:
I think that using ```implode``` would had been better. (Like it had been done in other places in the same file)

Reply:
This is done in #7174